fe/fe-core/src/main/java/com/starrocks/sql/optimizer/RequiredPropertyDeriver.java

```diff
@@ -177,7 +177,7 @@ public Void visitPhysicalHashAggregate(PhysicalHashAggregateOperator node, Expre
         // If scan tablet sum leas than 1, do one phase local aggregate is enough
         if (ConnectContext.get().getSessionVariable().getNewPlannerAggStage() == 0
                 && context.getRootProperty().isExecuteInOneTablet()
-                && node.getType().isGlobal() && !node.isSplit()) {
+                && node.isOnePhaseAgg()) {
```

Comment:
OnePhaseAgg may be split, too.  is it safety here?

Reply:
onePhaseAgg cannot be split, they are identical.
```java
    public boolean isOnePhaseAgg() {
        return type.isGlobal() && !isSplit;
    }
```